devito/ops/compiler.py

```diff
@@ -0,0 +1,74 @@
+from codepy.jit import compile_from_string
+from time import time
+import warnings
+
+from devito.logger import debug
+from devito.parameters import configuration
+
+
+__all__ = ['CompilerOPS']
+
+
+class CompilerOPS(configuration['compiler'].__class__):
+    def __init__(self, *args, **kwargs):
+        super(CompilerOPS, self).__init__(*args, **kwargs)
+
+    def jit_compile_ops(self, soname, code, hcode):
```

Comment:
you're not exploiting inheritance, I don't like this

Reply:
How so? Could you comment further?

Reply:
you're not overriding ``jit_compile`` and/or calling it via ``super``

you're just copy-pasting, which is of course bad